repology/parsers/parsers/sisyphus.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (C) 2021 Danil Shein <dshein@altlinux.org>
+#
+# This file is part of repology
+#
+# repology is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# repology is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with repology.  If not, see <http://www.gnu.org/licenses/>.
+
+from typing import Iterable
+
+from repology.packagemaker import LinkType, NameType, PackageFactory, PackageMaker
```

Comment:
LinkType comes from `repology.package`, that's the cause for build failure.

Reply:
done